content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -410,20 +341,6 @@ nodeRegistration:
 Alternatively, you can modify `/etc/fstab` to make the `/usr` mount writeable, but please
 be advised that this is modifying a design principle of the Linux distribution.
 
-## `kubeadm upgrade plan` prints out `context deadline exceeded` error message
-
-This error message is shown when upgrading a Kubernetes cluster with `kubeadm` in the case of running an external etcd. This is not a critical bug and happens because older versions of kubeadm perform a version check on the external etcd cluster. You can proceed with `kubeadm upgrade apply ...`.
-
-This issue is fixed as of version 1.19.
-
-## `kubeadm reset` unmounts `/var/lib/kubelet`
-
-If `/var/lib/kubelet` is being mounted, performing a `kubeadm reset` will effectively unmount it.
-
-To workaround the issue, re-mount the `/var/lib/kubelet` directory after performing the `kubeadm reset` operation.
-
-This is a regression introduced in kubeadm 1.15. The issue is fixed in 1.20.
-
```

Comment:
How come we're removing this here and not in a different PR?

Reply:
https://github.com/kubernetes/website/pull/31309#discussion_r789131314